### PR TITLE
Add expressRouteGatewayBypass for enabling FastPath to vWan Firewall hub

### PR DESCRIPTION
**Pull request by sachethegde**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by lirenhe** [comment]
@sachethegde, the branch created in for July release, but it looks you are modifying 2020-06-01 version, is this expected?

**Comment by sachethegde** [comment]
@lirenhe Updated the Target version to July

File: specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualWan.json [hunk 1]
```diff
@@ -6884,6 +6884,10 @@
           "type": "boolean",
           "description": "Enable internet security."
         },
+        "expressRouteGatewayBypass": {
+          "type": "boolean",
+          "description": "Enable FastPath to vWan Firewall hub."
```
Comment (allenjzhang) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualWan.json:6889]: seeing all examples are at false, do you want to add `"default": false`

**Review by allenjzhang — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualWan.json (hunk 1 above)
Comment (sachethegde) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualWan.json:6889]: I have kept the default setting same as this flag used in a different resource. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/virtualNetworkGateway.json#L2986
Let me get back on this.

Comment (sachethegde) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualWan.json:6889]: @allenjzhang Keeping the setting as the flag used in the other resource. We can merger this now 👍

**Pull request merged**